wordpress-mobile/WordPress-Android
# Login rework: Epilogue screen

Introduces the login epilogue screen that shows the newly logged in user and their sites list.

The epilogue is shown when the magiclink flow concludes successfully (the deeplink is intercepted).

The final visual touches, the messages and final texts will be done in a future PR.

Of note:
* The epilogue screen is mainly informational. User can dismiss it or tap "Continue" and will still be logged in.
* The "Connect another site" button is not implemented yet.
* The deeplink is handled directly via `WPMainActivity`
* The auth token is saved in `WPMainActivity` but the account, account settings and sites list messages are emitted in the epilogue screen so the progress bar in the epilogue can catch the sites update finishing

To test:
* With the app data clean, follow the login magiclink flow
* At the "magic link sent" screen, the "Open mail" button is not yet implemented so, open your email client manually and follow the instructions of the magiclink email. Upon validation of the magiclink, the app should intercept the deeplink and be brought to the foreground
* The epilogue screen should be shown and among other things, it should display your sites list
* If the sites list is big enough, you might notice the progress bar spin while updating and disappear when finished
* Tap "Continue" or back to dismiss the screen and return to the main app
